app/lib/forms/question_types/check_box.rb

```diff
@@ -1,30 +1,22 @@
 module Forms
   module QuestionTypes
     class CheckBox < Base
-      DEFAULT_STYLES = {
-        legend: {
-          size: "xl",
-          tag: "h1",
-        }.freeze,
-        hint: nil,
-      }.freeze
-
-      attr_reader :checked_value, :unchecked_value, :required, :body
-
-      def initialize(*args, checked_value: "1", unchecked_value: "0", required: false, **opts)
-        @checked_value = checked_value
-        @unchecked_value = unchecked_value
-        @required = required
-
-        super(*args, **opts)
-      end
+      attribute :checked_value, default: "1".freeze
+      attribute :unchecked_value, default: "0".freeze
```

Comment:
Just my 2 cents but maybe it would be better to use constants or symbols here?

Something like


```
CHECKED_VALUE = "1".freeze
  UNCHECKED_VALUE = "0".freeze

  attribute :checked_value, default: CHECKED_VALUE
  attribute :unchecked_value, default: UNCHECKED_VALUE
```

Or

`attribute :checked_value, default: :checked`
`attribute :unchecked_value, default: :unchecked`

Reply:
OOI - why would it be better? Those are only the default value of dynamic state. The values are set to match default values of gov.uk component that is being used. I don't really see any benefit of using constants here as those values should not be used anywhere without the instances. Changing default values to :checked and :unchecked will break rails params casting.